wikidict/lang/en/template_handlers.py

```diff
@@ -559,6 +559,8 @@ def render_morphology(tpl: str, parts: List[str], data: Dict[str, str]) -> str:
     '<i>toto</i>&nbsp;+&nbsp;<i>-alt2</i> (<i>-tr2</i>, “t2”, pos2)'
     >>> render_morphology("prefix", ["en", "toto", "lala"], defaultdict(str, {"t2":"t2", "tr2":"tr2", "alt2":"alt2", "pos2":"pos2"}))
     '<i>toto-</i>&nbsp;+&nbsp;<i>alt2</i> (<i>tr2</i>, “t2”, pos2)'
+    >>> render_morphology("suffix", ["en", "", "cide"], defaultdict(str))
+    '<i></i>&nbsp;+&nbsp;<i>cide</i>'
```

Comment:
Could we get rid of the empty tags at the start of the line?

Reply:
I will have another look tomorrow :) I close the PR